articles/112_generated_interfaces_cpp.md

```diff
@@ -101,6 +112,9 @@ For each pointer type there a non-const and a const `typedef`:
 * `UniquePtr` and `ConstUniquePtr`
 * `WeakPtr` and `ConstWeakPtr`
 
+<div class="alert alert-warning" markdown="1">
+  **Note:** `Ptr` and `ConstPtr` are *not* compatible with ROS 1 usage (which is shared). That may change.
+</div>
```

Comment:
I prefer these typedefs proposed by @scpeters, instead:

* `RawPtr` and `ConstRawPtr`
* `Ptr` and `ConstPtr`
* `UniquePtr` and `ConstUniquePtr`
* `WeakPtr` and `ConstWeakPtr`